### DISCUSSION TITLE
[회의록] : 2주차 김율민

### DISCUSSION THREAD

**Discussion by yulmai999** [FE3-part4-team2/-Nomad#26] [회의록]
### ✏️안건(회의내용)
- alert창을 toast 처리 하는것에 대한 논의
- rem 이 제대로 적용되지않는문제
   ㄴ> :root 에 fontsize 지정해서 해결
- 컴포넌트 활용문제
  ㄴ> ...rest를 사용해서 좀더 단순하게 컴포넌트에 프롭스를 전달해서 스타일을 조정할수있게함.
- 체험 등록 후 어떻게 이동할지에 대한 논의
  ㄴ>내 체험 등록이 성공적으로 완료되면 내 체험 관리 페이지로 이동 후 toast로 알림

#### 메모해둔내용이 날아가서 기억나는것만 적었습니다. 추가하실내용 있으시면 추가해주세요

**Comment by jjunhyuki** [comment]
# 24.04.24.수

# ✏️ 안건 
- 벌써 플젝 진행한지 2주가 지났는데 우리 모두 화이팅을 해야할듯!

# ✏️진행상황 
- 다들 issue에 올려둔 페이지에서 css 애를 먹고있다.

# ✏️메모
- 지희님 다음주 목,금 (5월 2,3일) 휴가

**Comment by yulmai999** [comment]
## 24.04.25.목

### ✏️안건(회의내용)
- 폴더구조 container 와 component를 확실히 구분해서 정리하자
  ㄴ component는 ui적인 부분을 담당하고 container는 데이터관리,함수처리 등 component에 필요한 상태관리를 담당.

참고자료
https://velog.io/@tree0787/component%ED%8C%8C%EC%9D%BC%EA%B3%BC-container%ED%8C%8C%EC%9D%BC%EC%9D%98-%EC%B0%A8%EC%9D%B4

**Comment by yulmai999** [comment]
## 24.04.26.금

### ✏️안건(회의내용)
- 재사용 가능한 input 컴포넌트 만드는중
- 일요일까지 회고작성 하기
- 중간점검 수요일 까지. 월요일 팀미팅에 자체점검
- axios instance 만들어야함
- pr  자주 올리기
